pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -2069,6 +2069,7 @@ var _ = Context("Scheduling", func() {
 				// delete the pod so that the node is empty
 				ExpectDeleted(ctx, env.Client, initialPod)
 				// startup taint + node not ready taint = 2
+				// note that this node is not post registration so the unregistered taint is present
```

Comment:
Is the unregistered taint still present or are you no longer applying it? Looks like the length is now still 2.